docs/src/tutorial/running.md

```diff
@@ -3,21 +3,21 @@
 >This section focuses on how to running the basic example build to give you a chance to practice and get a sense of how things work. The next section covers customizing and configuring your own build.
```

Comment:
This page feels really out of place - the layout is different from the preceeding 3 tutorials, it uses config files in `./my_profiles` not `./ncov-tutorial`, and doesn't really cover any new ground. I'd suggest removing the tutorial part, but keeping the "Troubleshoot common issues" part.

Reply:
This was retained in 8c7a4115ee215a4a3a54260b1e3dc040be61c216 but I have yet to update the contents.

Reply:
Done: 5a45a987